typescript-babel-combination/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "example-typescript-babel-combination",
+  "version": "1.0.0",
+  "description": "TypeScript-Babel Combination",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "scripts": {
+    "check-types": "tsc",
```

Comment:
Should it have a `tslint` config file?

Reply:
I forgot. I have just added `tslint` setup.